src/metabase/upload.clj

```diff
@@ -104,12 +138,22 @@
   "All type tags that correspond to concrete column types."
   (into #{} (remove abstract->concrete) value-types))
 
+(defn- column-type?
+  [value-type]
+  (contains? column-types value-type))
+
 (defn ^:private column-type
   "The most specific column type corresponding to the given value type."
```

Comment:
This docstring needs updating. It's now more complicated than this because of line 154.
https://github.com/metabase/metabase/pull/40275/files#diff-af3188166dbf14ec61d196f0e71de90a50f7e2c9de2c152d7cd2e20c6af4051bR154

It should mention that the value can be coerced to an `existing-type` if the value-type is coercible.